daktari/checks/intellij_idea.py

```diff
@@ -160,3 +161,38 @@ def validate_query_result(self, result):
         current_package_manager = str(key_json["keyToString"]["nodejs_package_manager_path"])
         logging.debug(f"IntelliJ node package manager set to: {current_package_manager}")
         return current_package_manager.__contains__(self.package_manager_path)
+
+
+class IntelliJProjectSdkJavaVersion(XmlFileXPathCheck):
+    name = "intellij.jdkVersionConfigured"
+    file_path = ".idea/misc.xml"
```

Comment:
Dunno if it's too much faff, but would it be worth a unit test that looks at a real `misc.xml` that we include in the repo?

Reply:
Yeah, not a faff at all I don't think - can sort out some tests for it tomorrow :+1: